src/integration-test/java/com/commercetools/project/sync/ProductSyncWithReferencesIT.java

```diff
@@ -198,9 +193,11 @@ void run_WithAProductWith500DistinctReferences_ShouldSyncCorrectly() {
       assertProductSyncerLoggingEvents(syncerTestLogger, 501);
       assertInventoryEntrySyncerLoggingEvents(syncerTestLogger, 0);
       assertCartDiscountSyncerLoggingEvents(syncerTestLogger, 0);
+      assertStateSyncerLoggingEvents(
+          syncerTestLogger, 1); // 1 state is built-in and it cant be deleted
 
-      // Every sync module (6 modules) is expected to have 2 logs (start and stats summary) = 12
-      assertThat(syncerTestLogger.getAllLoggingEvents()).hasSize(12);
+      // Every sync module (7 modules) is expected to have 2 logs (start and stats summary) = 12
```

Comment:
7 * 2 === 12? 🐙

Reply:
It's better to do a full project search from a portion of this comment and look everywhere this has changed properly :)

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/446caae5aacd2064b4580790e3c944b97110e27b